frontend/src/feature/cards/components/CardDetails.vue

```diff
@@ -41,7 +41,7 @@
       <v-spacer/>
       <edit-mdi-button tooltip-text="Edit Card"
                        :click-handler="() => emit('edit', true)"/>
-      <delete-mdi-button :click-handler="() => {}" tooltip-text="Delete Card - not implemented"/>
+      <delete-mdi-button :click-handler="() => emit('delete', card)"/>
```

Comment:
Pls add tooltipText property - for convenience, they are on all the buttons:
`tooltip-text="Delete Card"` is sufficient